portals/publisher/src/main/webapp/source/src/app/data/AuthManager.js

```diff
@@ -185,6 +185,22 @@ class AuthManager {
         }
     }
 
+    /**
+     *
+     * Check whether user is a admin user
+     * @static
+     * @returns {Boolean} isAdminUser
+     * @memberof AuthManager
+     */
+    static isAdminUser() {
+        if (AuthManager.getUser() === null) {
```

Comment:
You can write the same as follows
`return AuthManager.getUser() && AuthManager.getUser().scopes.includes ('apim:admin')`

Reply:
I have added this change